docker/cisco-meraki/Pipfile

```diff
@@ -0,0 +1,20 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+"ruamel.yaml" = "*"
+requests = "*"
+dockerfile_parse = "*"
+pyyaml = "*"
+regex = "*"
+python-dateutil = "*"
+gitpython = "*"
+meraki="1.46.0"
```

Comment:
Change it to be `"*"` or add a comment explaining why we are closing this specific version.

Reply:
No particular reason, version 1.46.0 was the version I was testing with